webofneeds/won-core/src/main/java/won/protocol/util/linkeddata/CachingLinkedDataSource.java

```diff
@@ -292,6 +311,9 @@ private DatasetResponseWithStatusCodeAndHeaders fetchAndCacheIfAppropriate(final
             if (expires != null && expires.getTime() == 0) {
                 // the expires header was invalid (e.g. '0'), which means: already expired.
                 // Don't cache.
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Fetched {}. Will not be cached due to invalid 'Expires' header sent by server");
```

Comment:
missing a parameter (and if clause is not necessary as long as the param wouldnt be calculated within the log statement itself)

Reply:
fixed, thanks